apps/test-quadrature64cpr/src/main.cpp

```diff
@@ -0,0 +1,35 @@
+#include "Logger.h"
+#include "Quadrature64CPR.h"
+
+void read_all();
+
+// first pin is yellow wire
+// second pin is white wire
+constexpr Encoder::Quadrature64CPR::Config config = {PA_5, PA_6, NC, 0};
+Encoder::Quadrature64CPR enc(config);
+
+Timer theta_timer, theta_dot_timer;
+float theta     = 0;
+float theta_dot = 0;
+
+int main() {
```

Comment:
Could you update the test app to have separate update and read threads? Check out the aeat-6012 test app for reference

Reply:
ive updated my code can you take a look?

Reply:
still havent had the chance to test it on the motor i will try to get to that soon